src/config/elasticsearch.ts

```
@@ -0,0 +1,8 @@
+import { Client } from "@elastic/elasticsearch";
+
+export const client = new Client({
+  cloud: {
+    id: process.env.CLOUD_ID || "",
```

we should throw an error if these env's are not set